Sources/Transaction/TransactionBuilder.swift

```diff
@@ -52,7 +52,8 @@ final class TransactionBuilder {
                             tombstoneBlockIndex,
                             fogResolverPtr,
                             memoBuilderPtr,
-                            blockVersion)
+                            blockVersion,
+                            nil)
```

Comment:
whats this change for ?

Reply:
the header file indicates that this parameter is `McError* MC_NULLABLE * MC_NULLABLE out_error`

so should be handled

Reply:
https://github.com/mobilecoinofficial/libmobilecoin-ios-artifacts/blob/release/v2/Artifacts/include/transaction.h#L193

Reply:
we need this change https://github.com/mobilecoinofficial/MobileCoin-Swift/pull/178